code/modules/surgery/organs/internal/heart/heart_anomalock.dm

```diff
@@ -7,9 +7,9 @@
 	name = "voltaic combat cyberheart"
 	desc = "A cutting-edge cyberheart, originally designed for Nanotrasen killsquad usage but later declassified for normal research. Voltaic technology allows the heart to keep the body upright in dire circumstances, alongside redirecting anomalous flux energy to fully shield the user from shocks and electro-magnetic pulses. Requires a refined Flux core as a power source."
 	icon_state = "anomalock_heart"
+	beat_noise = "an astonishing <b>BZZZ</b> of immense electrical power"
 	bleed_prevention = TRUE
 	toxification_probability = 0
-
 	COOLDOWN_DECLARE(survival_cooldown)
```

Comment:
a bunch of these empty lines are, in fact, intentionally added to help readability
so please put this one back